interchain/scripts/src/abstract_ibc.rs

```diff
@@ -48,3 +50,30 @@ pub fn abstract_ibc_connection_with<Chain: IbcQueryHandler, IBC: InterchainEnv<C
 
     Ok(())
 }
+
+pub fn get_polytone_deployment_id(src_chain: &ChainInfo, dst_chain: &ChainInfo) -> String {
+    format!("{}-->{}", src_chain.chain_id, dst_chain.chain_id)
+}
+pub fn has_polytone_connection(
+    src_chain: ChainInfo,
+    dst_chain: ChainInfo,
+    rt: &Handle,
+) -> anyhow::Result<bool> {
+    // We just need to verify if the polytone deployment crate has the contracts in it
```

Comment:
What if there is no ibc channel tho?

Reply:
What do you mean ?

Reply:
Like Polytone can be deployed on both chains, but no connection between them, or is it not possible? Or is it covered by deployment id?

Reply:
In that case they wouldn't be in here afaik